dll/win32/iernonce/iernonce.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Functions to read RunOnceEx registry.
+ * COPYRIGHT:   Copyright 2013-2016 Robert Naumann
+ *              Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+
+#define WIN32_NO_STATUS
+#include <windef.h>
+#include <winbase.h>
+#include <windows.h>
+
+#define NDEBUG
+#include <debug.h>
+
+#include "registry.h"
+
+
+HINSTANCE hInstance;
+
+BOOL
+WINAPI
+DllMain(HINSTANCE hinstDLL,
+        DWORD dwReason,
+        LPVOID reserved)
+{
+    switch (dwReason)
+    {
+        case DLL_PROCESS_ATTACH:
+            break;
+        case DLL_PROCESS_DETACH:
+            hInstance = hinstDLL;
+            break;
+    }
```

Comment:
I don't know why, but this is here since I start to work on `iernonce`, and it seems not being used at all 🤣
I guess this is a mistake, and I think removing it is better?

Reply:
The general rule of the thumb is that when a DLL is attached to the process' private address space, you'd assign a handle instance from the DLL instance point itself, and not when the DLL is detached from the process.